test/verify/test_scatternd.cpp

```diff
@@ -0,0 +1,28 @@
+#include "verify_program.hpp"
```

Comment:
Can we change one or two of these verify tests to a non-standard input shape?, like one in transposed, another is in broadcasted?